pkg/azuredisk/azuredisk_v2.go

```diff
@@ -278,16 +296,22 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 
 	// Setup a new controller to clean-up AzDriverNodes
 	// objects for the nodes which get deleted
-	klog.V(2).Info("Initializing AzDriverNode controller")
-	err = controller.InitializeAzDriverNodeController(mgr, &d.azDiskClient, d.objectNamespace)
+	// klog.V(2).Info("Initializing AzDriverNode controller")
+	// // err = controller.InitializeAzDriverNodeController(mgr, &d.azDiskClient, d.objectNamespace)
+	// // if err != nil {
+	// // 	klog.Errorf("Failed to initialize AzDriverNodeController. Error (%v). Exiting application...", err)
+	// // 	os.Exit(1)
+	// // }
```

Comment:
Why is this commented out?

Reply:
I commented it out to test just azvolumeattachment controller alone. I commented it back in. :)